### PR TITLE
Updated oracle logo

### PR DESCRIPTION
**Pull request by poixen**
Updated logo

### CODE REVIEW

**Comment by poixen** [comment]
Should fix #665

File: oracle/resources/oracle.svg [hunk 1]
```diff
@@ -176,53 +270,154 @@
...
+       style="fill:#c87137;fill-opacity:1;fill-rule:nonzero;stroke:#000000;stroke-opacity:0"
+       id="path5461-8"
+       sodipodi:cx="-84.135757"
+       sodipodi:cy="155.38211"
+       sodipodi:rx="69.231705"
+       sodipodi:ry="144.23273"
+       d="m -14.904053,155.38211 c 0,79.65754 -30.99609,144.23273 -69.231704,144.23273 -38.235613,0 -69.231703,-64.57519 -69.231703,-144.23273 0,-79.657535 30.99609,-144.232726 69.231703,-144.232726 38.235614,0 69.231704,64.575191 69.231704,144.232726 z"
+       transform="matrix(1.228342,0,0,0.85746409,211.0465,16.713831)" />
+    <g
+       style="font-size:456.36495972000000000px;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;line-height:125%;letter-spacing:0px;word-spacing:0px;fill:#ffffff;fill-opacity:1;stroke:none;font-family:Adobe Devanagari;-inkscape-font-specification:Adobe Devanagari"
```
Comment (ctrlaltca) [inline oracle/resources/oracle.svg]: Hum this smells: ```font-family:Adobe Devanagari;-inkscape-font-specification:Adobe Devanagari```

Comment (poixen) [inline oracle/resources/oracle.svg]: I have now vacuumed the defs, though the font is still there, I guess it shows what the path was created from, it shouldnt be an issue. Did you get to try it out?

Comment (ctrlaltca) [inline oracle/resources/oracle.svg]: Yes, and it seems to work fine, so i suppose it's just a leftover of inkspace to know what the path was created from as you suggested. :+1:

**Comment by Daenyth** [comment]
Icon looked fine on mac in DMG, launcher, and dock.

**Comment by poixen** [comment]
This good to merge now?

**Comment by ctrlaltca** [comment]
The png version (actually used only under linux) has changed its size from 48x48 to 34x48. I don't know if this can create problems with some window manager..

**Comment by ZeldaZach** [comment]
I think we should add white space to make it 48x48 again..

**Comment by poixen** [comment]
Updated to be 48x48

**Comment by ctrlaltca** [comment]
It seems everything's fine now, merge++ :+1:

**Pull request merged**